src/main/java/hudson/plugins/octopusdeploy/services/StringUtil.java

```diff
@@ -0,0 +1,9 @@
+package hudson.plugins.octopusdeploy.services;
+
+import static org.apache.commons.lang.StringUtils.trim;
+
+public class StringUtil {
+    public static String cleanValue(String value) {
```

Comment:
Would `sanitizeParameterValue` be more explicit?

Reply:
👍